EE_API_automation/pytest/runQuickStart.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+# Do not reveal secrets
+set +x
+
+JENKINSLOG=$(pwd)/pytest_junit_logs.xml
+CLILOG=$(pwd)/pytest_cli_logs.log
+echo ""
+echo Using logfile $JENKINSLOG and $CLILOG
+echo ""
+if test ! $(which pytest)
+    then
+        echo "==> Check pre-requisites: python and pytest should be installed"
+else
+    echo "==> Running GettingStarted API tests"
+    if [ $# -lt 3 ]
+    then
+        echo "==> Insufficient arguments provided! Need minimum 3. Exiting."
+        exit
+
+    elif [ -z "$4" ] || [ "$4" = "False" ] || [ "$4" = "false" ]
+    then
+        echo "==> cleanup=True flag not supplied. ***** SPACES WILL BE PERSISTED!!! *****"
```

Comment:
You don't want to either create or persist a space, right?

Reply:
Also, don't remove the option to run just the API tests (the condition where UI is not involved) as it may be needed later.

Reply:
@rgarg1 we can add this later when we have many tests and more than 1 file. For now, I think it is fine.
WDYT?